website/docs/reference/resource-configs/spark-configs.md

```diff
@@ -191,19 +191,22 @@ insert overwrite table analytics.spark_incremental
 
 
 **Usage notes:** The `merge` incremental strategy requires:
-- `file_format: delta`
-- Databricks Runtime 5.1 and above
+- `file_format: delta or hudi`
+- Databricks Runtime 5.1 and above for delta file format
+- Apache Spark for hudi file format
```

Comment:
naive question do Databricks Spark clusters support Hudi?

Reply:
We haven't tested Hudi on Databricks Spark cluster, theoretically, it should work unless there are any specific differences from vanilla spark runtime. To be on the safe side, we are recommending users use only Spark runtime.